application/error_handlers.py

```diff
@@ -0,0 +1,27 @@
+import flask
```

Comment:
Can we use pep8 standards for import statements - https://www.python.org/dev/peps/pep-0008/#imports